frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -321,6 +305,19 @@ class Sidebar extends Component<Props> {
                             <span className="sidebar_label">{t('sidebar.settings')}</span>
                         </Link>
                     </div>
+                    {(debug && keystores.some(({ type }) => type === 'software') && deviceIDs.length === 0) && (
```

Comment:
why is `deviceIDs.length === 0` needed? is there a software keystore with non-empty device ID which you wouldn't want to "eject"?